src/pinning.js

```diff
@@ -236,6 +237,13 @@ class Pinning {
       } else if (data.type === 'SYNC_DB' && data.thread) {
         this.openDB(data.odbAddress, this._sendHasResponse.bind(this))
       }
+      if (data.did) {
+        // We resolve the DID in order to pin the ipfs object
+        try {
+          resolveDID(this.ipfs, data.did)
```

Comment:
so this pins it because the client and server node are connected at this point, and this will fetch the object ?

Reply:
exactly yes